### DISCUSSION TITLE
compatible mode

### DISCUSSION THREAD

**Discussion by Zcamaya** [weihuoya/citra#777] [Q&A]
what does this thing do?
![Screenshot_2023-04-24-19-03-33-78_c198c715d99ba250d5a335743408f64f](https://user-images.githubusercontent.com/114555605/233979438-ed7ff7a1-bb59-48d9-8208-75ca5ad402bf.jpg)

**Comment by Tappy-tech** [comment]
For now, it's best to await Weihuoya's answer, as nobody else, even on the reddit post (Except me) responded to a similiar question. And I don't think we can expect someone else to give an answer to what it does. As for the reason why, I've made a comment about that in the Reddit post (More like an opinion on it). 
In a nutshell, it's mostly because most of the Citra MMJ community doesn't really use it (Proved by how people help each other and how most of the YouTubers just look at the changelog instead of opening an app and seeing changes), and some people who use it can't figure out how to solve a certain problem, and instead they just ask, but not search for a solution.

It can be speculated that it fixes some minor problems with graphics (If we take to the account Weibo and Github changelog)

**Comment by Deka710** [comment]
https://github.com/weihuoya/citra/issues/775#issuecomment-1517313600
